nengo_gui/static/components/value.js

```diff
@@ -56,6 +56,27 @@ Nengo.Value = function(parent, sim, args) {
     this.on_resize(this.get_screen_width(), this.get_screen_height());
     this.axes2d.axis_y.tickValues([args.min_value, args.max_value]);
     this.axes2d.fit_ticks(this);
+
+    this.colors = Nengo.make_colors(6);
+    this.color_func = function(d, i) {return self.colors[i % 6]};
+    this.legend = document.createElement('div');
+    this.legend.classList.add('legend');
+    this.div.appendChild(this.legend);
+
+    this.legend_labels = args.legend_labels || [];
+    if(this.legend_labels.length !== this.n_lines){
+        // fill up an array with temporary labels
+        for(i=0; i<this.n_lines; i++){
+            if(this.legend_labels[i] === undefined){
+                this.legend_labels[i] = "label_".concat(String(i));
```

Comment:
Quick style question:  is there a preference to do ```"label_".concat(String(i))``` rather than ```"label_" + i```?  I find the latter clearer, but it's just what I'm used to.